src/mtp_cmd_handler.c

```diff
@@ -971,6 +971,11 @@ static void __set_object_protection(mtp_handler_t *hdlr)
 }
 #endif /* MTP_SUPPORT_SET_PROTECTION */
 
+static void __begin_end_edit_object(mtp_handler_t *hdlr)
+{
+	_cmd_hdlr_send_response_code(hdlr, PTP_RESPONSE_OK);
```

Comment:
So in essence the implementation does nothing except sending PTP_RESPONSE_OK?

Reply:
> So in essence the implementation does nothing except sending PTP_RESPONSE_OK?

Currently BeginEdit & EndEdit event is triggered before calling SendPartialObject and expects to return success, else the next subsequent SendPartialObject is not triggered.